docs/modules/quick-start/pages/situations.adoc

```diff
@@ -0,0 +1,51 @@
+
+:imagesdir: ../assets/images
+= Step 2: View and Create Situations
+
+Now that you have xref:engine-config.adoc[selected the Clustering engine], ALEC automatically begins to identify situations.
+This requires new alarms to be triggered within your OpenNMS environment, so--depending on your network's usual activity--this may take some time.
+
+== View situations
+
+After alarms have been triggered in OpenNMS, they are evaluated by the Clustering engine and grouped into clusters as appropriate.
+You can view current situations in the OpenNMS web UI: click *Plugins > ALEC* in the top menu bar.
+The Situation List is displayed:
+
+.ALEC Situation List page
+image::situation-list.png[ALEC Situation List page displaying active situations, 800]
+
+Active situations are sorted from most to least severe.
+You can modify the sorting method by using the filters at the left of the screen.
+To return to the default sorting and filtering settings, click *Reset Filters*.
+
+To view more information on a specific situation, click its entry in the list.
+The situation details page is displayed:
+
+.Situation details page
+image::situation-details.png[Situation details page for a critical situation that affects two nodes, 800]
+
+From this page, you can acknowledge the situation (indicate that it has been seen and is being addressed), clear it (mark the situation as not relevant), or reject it (indicate that the clustered alarms are not related).
+You can also take action on individual alarms within the cluster by selecting them in the *Alarms* list.
+
+You can create memos for both the situation overall and for individual alarms.
```

Comment:
What is the difference between a sticky memo and a journal memo?

Reply:
I'll clarify this as well.